README.md

```diff
@@ -8,14 +8,15 @@ __Programming language:__ JavaScript
 __Actions:__ Read, Watch, Buy  
```

Comment:
What lab is it? Or it just routine fix?

Reply:
It is Lab6. I added mobile adaptation and fixed bugs there.

Reply:
Previous pull request #6 it was lab7 - DB implementation